src/create-config.js

```diff
@@ -1,19 +1,18 @@
-'use strict';
+"use strict";
```

Comment:
@RusinovAnton Why double quotes?

Reply:
This was formatted with default Prettier settings in my editor. I would like to make separate PR that adds Prettier and our settings to this repo and format code then.